components/common/header/authNav.js

```diff
@@ -82,7 +82,8 @@ const AuthNav = ({ user, onLogout }) => {
             />
           </MenuButton>
           <MenuList backgroundColor='lightRock'>
-            <MenuItem
+            {/* TODO: commenting these out for now in case we decide to add them back, but I agree with Eryn that they aren't needed */}
```

Comment:
i'd prefer we have these, especially as our product is young

Reply:
also now there's no way to get back to the dashboard from the splash page (if a user gets there)

Reply:
i'd say leave the links in unless they are all in the Footer :D

Reply:
Reverted these changes.